src/main/java/org/cryptomator/cryptofs/DirectoryIdLoader.java

```diff
@@ -24,6 +25,7 @@ public DirectoryIdLoader() {
 
 	@Override
 	public String load(Path dirFilePath) throws IOException {
+		//TODO: replace by Files.readString(StandardCharsets.UTF_8)
```

Comment:
Behaviour is different in regards to
1. unexpected file size
2. thus memory consumption
3. handling of empty files
4. fallback in case of `NoSuchFileException`

Reply:
Correct, hence it is only a TODO. Acutally, it wass intended as a reminder. But since it is not related to this PR, i'll remove it.

Reply:
Fixed in [1f72a65](https://github.com/cryptomator/cryptofs/pull/118/commits/1f72a65ce639e8fe1fbd6e428bd6fb81268c0a1c)